src/helpers.js

```diff
@@ -105,11 +105,13 @@ function getWindowFromNode(node) {
     throw new Error(
       `It looks like you passed an Array instead of a DOM node. Did you do something like \`fireEvent.click(screen.getAllBy...\` when you meant to use a \`getBy\` query \`fireEvent.click(screen.getBy...\`?`,
     )
-  } else {
-    // The user passed something unusual to a calling function
+  } else if (node instanceof HTMLElement) {
     throw new Error(
       `Unable to find the "window" object for the given node. Please file an issue with the code that's causing you to see this error: https://github.com/testing-library/dom-testing-library/issues/new`,
     )
+  } else {
+    // The user passed something unusual to a calling function
+    throw new Error(`The given node is not an HTMLElement.`)
```

Comment:
I think we need to provide a bit more guidance here.
Could we add the type or the value of `node` to the message?